### DISCUSSION TITLE
Can we use Mica and Mica Alt at the same time in one window?

### DISCUSSION THREAD

**Discussion by ghost1372** [microsoft/WindowsAppSDK#3192] [Q&A]
Hi,
Since every Window initializes Mica once, can we use Mica and Mica Alt at the same time in one Windows?

**Comment by DarranRowe** [comment]
What do you mean by "at the same time"?

The meaning of "at the same time" in this case implies that you want Mica and Mica Alt to be active as the system backdrop at the exact same instance in time. This would mean either merging Mica and Mica Alt somehow, or having multiple controllers active for different regions of a window.

I honestly don't believe this is what you mean.

If you mean "are you able to use the same Mica controller to change between Mica and Mica Alt?" Then yes, the Mica controller has a Kind property that allows you to change between the two Mica kinds dynamically.

**Reply by ghost1372** [reply]
Tnx, i need multiple controllers active for different regions of a window.
Is it possible?

**Reply by DarranRowe** [reply]
I was looking around a bit just in case I missed something, but there is nothing exposed in the API that allows this.
From everything I have seen, the backdrop controller and the Xaml window places and sizes the backdrop based upon the assumption that it will be the only backdrop.
